common/src/calendar/__tests__/calendar.test.ts

```diff
@@ -658,7 +784,7 @@ describe("isReservationUnitReservable", () => {
         metadataSet: {
           supportedFields: ["name"],
         },
-      } as ReservationUnitByPkType)
+      } as unknown as ReservationUnitByPkType)
```

Comment:
Not a fan of casting out type safety even in tests.

Could we just add common stuff in the file / describe level and copy it to data? ex.

```
const commonParams = {
    allowReservationsWithoutOpeningHours: false,
    authentication: ReservationUnitsReservationUnitAuthenticationChoices.Weak,
    canApplyFreeOfCharge: false,
    contactInformation: "",
    isArchived: false,
    isDraft: false,
    requireIntroduction: false,
    requireReservationHandling: false,
    reservationKind:
      ReservationUnitsReservationUnitReservationKindChoices.Direct,
    reservationStartInterval:
      ReservationUnitsReservationUnitReservationStartIntervalChoices.Interval_15Mins,
    uuid: "",
  };

isReservationUnitReservable({
        ...commonParams,
        id: "1234",
        minReservationDuration: 3600,
        maxReservationDuration: 3600,
        reservationBegins: String(addMinutes(new Date(), -10)),
        openingHours,
        metadataSet: {
          id: "",
          name: "",
          supportedFields: ["name"],
        },
})
```

That goes through typecheck without any casts at least in VS code TS 5

Reply:
Good call, corrected.